tests/test-json/src/main.rs

```diff
@@ -176,29 +123,168 @@ mod tests {
                 }
                 _ => {}
             }
-            println!("DTrace not yet ready");
+            // println!("DTrace not yet ready");
```

Comment:
looks like this got commented out; are there other changes?

Reply:
Hmm, this seems like an oversight/mistake; I'll fix this and go over the code once more to make sure I've not left other such unrelated changes in.